website/docs/components/dropdown/partials/guidelines/guidelines.md

```diff
@@ -156,98 +156,186 @@ If you do not want the width of the List to expand automatically to accommodate
+</div>
+
+### Footer
+
+A Footer provides a fixed space at the bottom of the List. Typically, Footers house actions related to the ListItems, e.g., when found in a filtering pattern an "Apply" Button can be used to submit the selections.
+
+<div class="hds-dropdown__content">
+  <Doc::ListContainer class="hds-dropdown__list">
+    <Hds::Dropdown::ListItem::Checkbox @value="Planned">Planned</Hds::Dropdown::ListItem::Checkbox>
+    <Hds::Dropdown::ListItem::Checkbox @value="Policy override">Policy override</Hds::Dropdown::ListItem::Checkbox>
+    <Hds::Dropdown::ListItem::Checkbox @value="Policy checked">Policy checked</Hds::Dropdown::ListItem::Checkbox>
+    <Hds::Dropdown::ListItem::Checkbox @value="Cost estimated">Cost estimated</Hds::Dropdown::ListItem::Checkbox>
+    <Hds::Dropdown::ListItem::Checkbox @value="Errored">Errored</Hds::Dropdown::ListItem::Checkbox>
+    <Hds::Dropdown::ListItem::Checkbox @value="Fetched">Fetched</Hds::Dropdown::ListItem::Checkbox>
+    <Hds::Dropdown::Footer @hasDivider={{true}}>
+      <Hds::ButtonSet>
+        <Hds::Button @text="Apply filters" type="submit" />
+        <Hds::Button @text="Cancel" @color="secondary" />
+      </Hds::ButtonSet>
+    </Hds::Dropdown::Footer>
   </Doc::ListContainer>
 </div>
 
 ## ListItem
 
 ### Types
 
-![Dropdown ListItem types](/assets/components/dropdown/dropdown-listitem-types.png =449x*)
+For maximum flexibility, we offer a variety of ListItems options. 
 
-!!! Info
+#### Interactive ListItems
+
+Use Interactive ListItems for actions (buttons) or links.
+
+- Use `Interactive - Critical` for destructive actions. 
+- Use `Interactive - Action` for everything else.
 
-**A note on Loading**
+![Interactive ListItem types](/assets/components/dropdown/dropdown-listitem-types-interactive.png =449x*)
+
+#### Selection ListItems
+
+Selection ListItems allow the user to select one or more options within a Dropdown.
+
+- Use `Checkmark` for switching context, e.g., organization switchers, project switchers, etc. Use for single selection only, as it doesn't provide enough indication that multi-selection is possible. For multi-selection, use `Checkbox`. 
+- Use `Checkbox` for multi-selection within a form or larger filter pattern.
+- Use `Radio` for single selection within a form or larger filter pattern.
+
+![Selection ListItem types](/assets/components/dropdown/dropdown-listitem-types-selection.png =449x*)
+
+!!! Do
+
+Use `Checkmark` for context switching. 
+
+![Example of proper checkmark list items](/assets/components/dropdown/dropdown-interactive-contextswitcher-do.png =280x*)
+!!!
+
+!!! Do
+
+Use `Checkbox` and `Radio` for filtering.
+
+![Example of proper checkbox and radio list items](/assets/components/dropdown/dropdown-interactive-filter-do.png =768x*)
+
+!!!
+
+!!! Dont
+
+Don't use `Checkmark` instead of `Radio` in larger filter patterns.  
+
+![Example of incorrect checkmark list items](/assets/components/dropdown/dropdown-interactive-filter-dont.png =768x*)
```

Comment:
Not sure the first block in the image is necessary (it may even be misleading, seems you shouldn't do that). Maybe leave only the actual `don't` part?